### ISSUE TITLE
@std/esm instead of backpack

### ISSUE THREAD

**Issue by logaretm** [nuxt-community/express-template#91]
Hello, I love using this template in our work and I would like to discuss a possible improvement.

### Overview

[@std/esm](https://github.com/standard-things/esm) Allows the use of ES6 modules with full spec compliance, works with Node.js 4+

While backpack is a great tool to use ES6 with Node today, but it adds a build overhead which consumes even more resources during the build step.

std/esm is very small, very fast, compliant with Node.js modules syntax and can be configured to support `.js` files too, and whenever node makes the jump to ES6 it can be easily removed without affecting the existing code base much.

I have successfully managed to swap out backpack in favor of this one and it works great so far.

One issue is the file watching, we can either use pm2 or nodemon, but I haven't tested it yet so far. I believe this is the primary reason you guys wanted to use backpack. (it wasn't around in the early days of this template).

This is how the files would change:

```js
// index.js

// Add the loader.
require = require('@std/esm')(module, { esm: 'js', cjs: true });

// User can put additional setup here, like env loading from a .env file or etc.

module.exports = require('./app').default;
```

### tldr;

less build time, smaller footprint, near built in performance, easily swapped out once Node.js officially support ES6 modules out of the box.

What do you guys think? I can pr this.

<div align="right"><sub><em>This question is available on <a href="https://nuxtjs.cmty.io">Nuxt.js</a> community (<a href="https://nuxtjs.cmty.io/nuxt-community/express-template/issues/c71">#c71</a>)</em></sub></div>

**Comment by ishowman** [comment]
no reply still?
